### PR TITLE
Merge support for _pageupVersion

### PR DESCRIPTION
**Pull request by elexisvenator**
[OSC-872] Add support for _pageupVersion in PageUp.CldrPackager as expected to receive from @pageup/locale

Also:
 - Updated testing to test all build targets (.netStandard2 and .net461)

### CODE REVIEW

File: src/PageUp.CldrPackager/CldrData.cs [hunk 1]
```diff
@@ -16,6 +16,12 @@ public class CldrData
         /// </summary>
         public IReadOnlyList<CldrLocale> AvailableLocales => this.Tree.Locales.Items.ToArray();
 
+        /// <summary>
+        /// Gets or sets the pageup version metadata value.
+        /// </summary>
+        [ProtoMember(2, IsRequired = false)]
+        public string PageupVersion { get; set; }
```
Comment (waltaro) [inline src/PageUp.CldrPackager/CldrData.cs]: `PageUpVersion`

File: src/PageUp.CldrPackager/Tools/CldrJsonMetadata.cs [hunk 2]
```diff
@@ -4,5 +4,6 @@ internal class CldrJsonMetadata
     {
         public string CldrVersion { get; set; }
         public CldrLocale CldrLocale { get; set; }
+        public string PageupVersion { get; set; }
```
Comment (waltaro) [inline src/PageUp.CldrPackager/Tools/CldrJsonMetadata.cs]: `PageUpVersion`

File: src/PageUp.CldrPackager/Tools/JsonParsers/Schemas/Main.txt [hunk 3]
```diff
@@ -51,10 +51,11 @@
       "type": "object",
       "properties": {
         "_number": 				{ "type": "string" },
-        "_cldrVersion":			{ "type": "string" }
+        "_cldrVersion":			{ "type": "string" },
+        "_pageupVersion":		{ "type": "string" }
```
Comment (waltaro) [inline src/PageUp.CldrPackager/Tools/JsonParsers/Schemas/Main.txt]: `_pageUpVersion` ?

Comment (waltaro) [reply src/PageUp.CldrPackager/Tools/JsonParsers/Schemas/Main.txt]: Also add one more tab!

Comment (elexisvenator) [reply src/PageUp.CldrPackager/Tools/JsonParsers/Schemas/Main.txt]: source file is _pageupVersion, it was confusing :D

Comment (elexisvenator) [reply src/PageUp.CldrPackager/Tools/JsonParsers/Schemas/Main.txt]: Also a classic battle of tabs vs spaces. This file uses a mix of both. In vs the indentation looks fine, but just to be sure i have fixed tabs/spaces in the whole file.

**Pull request merged**